src/views/notifications/Create.vue

```diff
@@ -0,0 +1,323 @@
+                              :class="'form-control' + (this.validation_error?.color != null ? ' border-theme-6' : '')"
+                              placeholder="Enter notification reason"
+                              v-model="this.create_notification.color"
+                            />
+                            <div v-if="this.validation_error?.color != null" class="text-theme-6 mt-2 mb-4">
+                              {{ this.validation_error?.color[0] }}
+                            </div>
+                          </div>
+                        </div>
+                        <div class="col-span-12 xxl:col-span-6 mb-4">
+                          <div>
+                            <label class="form-label">
+                              Type
+                            </label>
+                            <TailSelect
+                              v-model="this.create_notification.type"
+                              :class="'form-control' + (this.validation_error?.type != null ? ' border-theme-6' : '')"
+                              :options="{
+                                search: true,
+                                classNames: 'w-full'
+                              }"
+                            >
+                              <option value=1>Icon</option>
+                              <option value=2>User</option>
+                            </TailSelect>
+                            <div v-if="this.validation_error?.type != null" class="text-theme-6 mt-2 mb-4">
+                              {{ this.validation_error?.type[0] }}
+                            </div>
+                            {{ this.create_notification.type }}
+                          </div>
+                        </div>
+                        <div class="col-span-12 xxl:col-span-12">
+                          <hr class="my-5">
+                        </div>
+                        <div class="col-span-12 xxl:col-span-6 mb-4">
+                          <div>
+                            <label for="create-icon-notification" class="form-label">
+                              Icon
+                            </label>
+                            <input
+                              id="create-icon-notification"
+                              type="text"
+                              :class="'form-control' + (this.validation_error?.icon != null ? ' border-theme-6' : '')"
+                              placeholder="Enter Icon"
+                              v-model="this.create_notification.icon"
+                              :disabled="parseInt(this.create_notification.type) !== 1"
+                            />
+                            <div v-if="parseInt(this.create_notification.type) !== 1" class="text-theme-6 mt-2 mb-4">
+                              User Type is selected!
+                            </div>
+                            <div v-if="this.validation_error?.icon != null" class="text-theme-6 mt-2 mb-4">
```

Comment:
Unnecessary `!= null`

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/34839e538cf6eea37658b6eebcf618108f1b8fc8